x/incentive/migrations/v9_migration.go

```diff
@@ -0,0 +1,11 @@
+package migrations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/incentive/types"
+)
+
+func (m Migrator) V9Migration(ctx sdk.Context) error {
+	m.keeper.SetParams(ctx, types.NewParams())
```

Comment:
Can you set NewParams to receive parameters?

Reply:
@jelysn i think we may have to rename the function to "DefaultParams())